.github/workflows/build-and-test.yaml

```diff
@@ -46,3 +46,4 @@ jobs:
           files: ${{ steps.test.outputs.coverage-report-files }}
           fail_ci_if_error: false
           verbose: true
+          flags: total
```

Comment:
I guess no flag means `total`? :thinking: Is it correct?

```suggestion
```

Reply:
A total flag is required when using the carry-forward flag feature.

Reply:
Reference https://docs.codecov.com/docs/carryforward-flags

Reply:
In that case, what's the difference between `project` and `project-total`?
![image](https://user-images.githubusercontent.com/31987104/161876642-7de3d99e-1f8f-496b-a0ff-08c23126589b.png)

Reply:
Also, if you delete the flag here, what happens? Will it raise an error? :thinking: 
I think the carry-forward feature isn't required here because all tests are executed.

Reply:
The `project` may be referring to a previous unflagged report.
In general, codecov tries to retain reports.

Reply:
Hmm, okay. Let's merge this and watch it for a while.

Reply:
> Also, if you delete the flag here, what happens? Will it raise an error? thinking I think the carry-forward feature isn't required here because all tests are executed.

No, no error occurs. However, the diff coverage reported in the PR comments will be incorrect.

> I think the carry-forward feature isn't required here because all tests are executed.

I think you misunderstand the carry-forward feature.

Reply:
Thank you! Merging.
Before that, I will attempt to explain a few things. (It would be best if Codecov's reference is helpful...)

### If there is no total flag
* codecov receives the diff coverage calculated by PR.
* The coverage computed in the base branch is not used. This is because the coverage with the carryforward flag does not exist in the base branch.
* Reports built on diff coverage will be incomplete.

### If the total flag is present
* codecov receives the diff coverage calculated by PR.
* It refers to the coverage computed in the base branch. Because the coverage with the carryforward flag exists in the base branch.
* codecov completes reports that are not present in the diff coverage with the base coverage.
* Reports created based on the coverage will be complete.